sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -108,6 +108,7 @@ private void InitializeBlobBindings(ExtensionConfigContext context)
 
             rule.BindToInput<MultiBlobContext>(this); // Intermediate private context to capture state
             rule.AddOpenConverter<MultiBlobContext, IEnumerable<BlobCollectionType>>(typeof(BlobCollectionConverter<>), this);
+            rule.AddOpenConverter<MultiBlobContext, BlobCollectionType[]>(typeof(BlobCollectionConverter<>), this);
```

Comment:
Need to add a string[] test to https://github.com/Azure/azure-sdk-for-net/blob/6b30cc4b715f7ee1e0f36c91285f3f445d50646d/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/samples/readmes/BlobExtensionSamples.cs#L261 for this

Reply:
added an array test

Reply:
Added a test for - `public void Run([Blob(ContainerName)] string[] blobs)`